bindings/python/src/operator.rs

```diff
@@ -184,15 +200,21 @@ impl Operator {
     }
 
     pub fn capability(&self) -> PyResult<capability::Capability> {
-        Ok(capability::Capability::new(self.0.info().full_capability()))
+        Ok(capability::Capability::new(
+            self.core.info().full_capability(),
+        ))
     }
 
     pub fn to_async_operator(&self) -> PyResult<AsyncOperator> {
-        Ok(AsyncOperator(self.0.clone().into()))
+        Ok(AsyncOperator {
+            core: self.core.clone().into(),
+            __scheme: self.__scheme.clone(),
+            __map: self.__map.clone(),
+        })
     }
 
     fn __repr__(&self) -> String {
```

Comment:
I think we need add extra __hash__ to represent two object are equal.

Reply:
It's hard to implement, we can't get internal information of `Operator`.

Reply:
how about hash the scheme and the map? 

__repr__ is used for show what the object is looking at. Maybe not a good method for identify

Reply:
I prefer don’t implement __hash__ and __eq__ for Operator, and remove the equal assertion based on __repr__.

(schema, map) can’t identify an Operator, leave __eq__ and __empty__ empty is better. Only the ID of PyObject can identify the object.

Reply:
> I prefer don’t implement **hash** and **eq** for Operator, and remove the equal assertion based on **repr**.

Agreed. `ocore::Operator` doesn't implement `Eq` ahd `Hash` too.